Directory.Build.targets

```diff
@@ -17,6 +17,10 @@
 
   <!-- Packaging Targets -->
   <PropertyGroup>
+    <!-- Generate snupkg package -->
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

Comment:
is this normal extension, not something like `symbols.nupkg`?

Reply:
We currently have `symbols.nupkg` but NuGet recently added support for portable pdbs via new packages called `.snupkg`s. Here is some additional info: https://devblogs.microsoft.com/nuget/improved-package-debugging-experience-with-the-nuget-org-symbol-server/

Basically the current guidance is that if you have a library that ships as a package in NuGet.org, you would want to also use NuGet's new .snupkg feature and push the packages there as well, for a better debugging experience.